src/rua_files.rs

```diff
@@ -21,6 +21,8 @@ pub struct RuaDirs {
 	global_checked_tars_dir: PathBuf,
 	/// Script used to wrap `makepkg` and related commands
 	pub wrapper_bwrap_script: PathBuf,
+	/// Script used to read `makepkg` config
```

Comment:
Now that the struct also contains paths to 2 files, should we
rename it to `RuaPaths`? (e.g. `rua_env.paths.review_dir()`)

Reply:
I would like to do it in a separate PR, if you don't mind.

Reply:
Sure, it's totally fine.